.coafile

```diff
@@ -6,3 +6,4 @@ bears = GitCommitBear
 
 [python]
 bears = PEP8Bear
+files = **.py
```

Comment:
as per gitmate result https://github.com/coala/coala-emacs/pull/18#issuecomment-317204432 maybe put the test files into an ignore?